cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -295,6 +295,25 @@ impl OpenshiftSecondaryMetadataParserPlugin {
         }
     }
 
+    async fn process_version(&self, data_dir: &PathBuf) -> Fallible<String> {
+        let supported_versions = ["1.0.0"];
```

Comment:
Lets make it a top-level `const`?

Reply:
This is the same public API vs. internal method discussion we're having in [this existing thread][1], right?

[1]: https://github.com/openshift/cincinnati/pull/314#discussion_r499784365

Reply:
Yes, however it would be easier to update this list ~~if~~ when we add new version there, even if its private

Reply:
Took a stab at this with 3b3e674c.